utils/s3_sync.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+
+def sync_open_dataset(dataset_dir):
+    os.makedirs(dataset_dir, exist_ok=True)
+    command = 'aws s3 sync "s3://autogpe-datasets/hdsm" "{}" --quiet'.format(dataset_dir)
+    print(command)
+    os.system(command)
+    print('===== Finished open datasets sync =====')
+
+
+def sync_airsim_dataset(dataset_dir):
+    os.makedirs(dataset_dir, exist_ok=True)
+    command = 'aws s3 sync "s3://autogpe-datasets/airsim_dataset" "{}/airsim" --quiet'.format(dataset_dir)
+    print(command)
+    os.system(command)
+    print('===== Finished airsim sync =====')
+
+
+def sync_lidar_dataset(dataset_dir):
+    os.makedirs(dataset_dir, exist_ok=True)
+    command = 'aws s3 sync "s3://autogpe-datasets/lidar-hdsm-dataset" "{}/lidar" --quiet'.format(dataset_dir)
```

Comment:
I think we may want to launch like a thread or process per pool and sync the top level directories for speed